jinahub/indexers/cache/__init__.py

```diff
@@ -105,28 +112,40 @@ def hash_doc(doc: Document, fields: Tuple[str]) -> bytes:
         return digest
 
     @property
-    def size(self):
-        assert len(self.cache_handler.id_to_hash) == len(
-            self.cache_handler.hash_to_id
-        ), 'Cache state invalid. Length of mappings does not align. Try rebuilding the index'
+    def ids(self):
```

Comment:
The naming seems to be confusing. Perhaps `ids_count` if `size` is no option

Reply:
I renamed them